bridge/test/nftbridge/SideNFTToken_test.js

```diff
@@ -1,6 +1,8 @@
 const SideToken = artifacts.require("./SideNFTToken");
 const utils = require("../utils");
 const truffleAssert = require("truffle-assertions");
+const keccak256 = web3.utils.keccak256;
+const ethUtil = require('ethereumjs-util');
```

Comment:
we're not using these right? I'd maybe remove them

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/268/commits/9c4e1c7b64b7379ff804bb3f768eed9c96a7461b